pkg/kubelet/lifecycle/predicate.go

```diff
@@ -221,33 +206,20 @@ func (e *PredicateFailureError) GetReason() string {
 	return e.PredicateDesc
 }
 
-// GeneralPredicates checks a group of predicates that the kubelet cares about.
-func GeneralPredicates(pod *v1.Pod, nodeInfo *schedulerframework.NodeInfo) ([]PredicateFailureReason, error) {
-	if nodeInfo.Node() == nil {
-		return nil, fmt.Errorf("node not found")
-	}
-
+// generalPredicates checks a group of predicates that the kubelet cares about.
+func generalFilters(pod *v1.Pod, nodeInfo *schedulerframework.NodeInfo) []PredicateFailureReason {
```

Comment:
generalFilters -> generalFilter (to be consistent with the one in eventhandlers.go)